### DISCUSSION TITLE
你好 我非常喜欢这篇文章 请问什么时候公布代码 我想尽快学习这篇文章

### DISCUSSION THREAD

**Discussion by wwwzztt** [PieceZhang/MPT-CataBlur#1] [General]
可以私信代码给我吗 我的联系方式是2317900307@qq.com 非常谢谢

**Comment by PieceZhang** [comment]
> 可以私信代码给我吗 我的联系方式是[2317900307@qq.com](mailto:2317900307@qq.com) 非常谢谢

Hi,
Code and dataset will be released as early as May or June. But I can't guarantee the time. Sorry about that.

**Comment by Zhang-bd** [comment]
你好，请问代码什么时候开源？
